### PR TITLE
Upgrade to kehaar 0.8.1

### PR DESCRIPTION
**Pull request by tie-rack**

### CODE REVIEW

File: resources/leiningen/new/kraken_http_api/channels.clj [hunk 1]
```diff
@@ -1,10 +1,6 @@
 (ns {{name}}.channels
   (:require [clojure.core.async :as async]))
 
-
-(defonce ok-requests (async/chan))
-(defonce ok-responses (async/chan))
-
 (defn close-all! []
-  (doseq [c [ok-requests ok-responses]]
+  (doseq [c []]
```
Comment (cap10morgan) [inline resources/leiningen/new/kraken_http_api/channels.clj]: Should we add a comment here about what this is for and that it should be removed if you don't end up needing it?

File: resources/leiningen/new/kraken_http_api/project.clj [hunk 2]
```diff
@@ -15,7 +15,7 @@
                  ;; kehaar.core/go-handler. (This may not be true anymore in
                  ;; core.async 0.2.x; need to test.)
                  [org.clojure/core.async "0.2.374"]
-                 [democracyworks/kehaar "0.5.0"]
+                 [democracyworks/kehaar "0.8.1"]
 
                  [io.pedestal/pedestal.service "0.4.1"]
                  [io.pedestal/pedestal.service-tools "0.4.1"]
```
Comment (cap10morgan) [inline resources/leiningen/new/kraken_http_api/project.clj:21]: Can we update the rest of the deps (that don't require other changes) while we're in here? Fine if you want to do it in another PR.

**Review by cap10morgan — CHANGES_REQUESTED** [review]

File: resources/leiningen/new/kraken_http_api/channels.clj (hunk 1 above)
Comment (tie-rack) [reply resources/leiningen/new/kraken_http_api/channels.clj]: Good idea.

File: resources/leiningen/new/kraken_http_api/project.clj (hunk 2 above)
Comment (tie-rack) [reply resources/leiningen/new/kraken_http_api/project.clj:21]: I vote to do it in another PR.

Comment (cap10morgan) [reply resources/leiningen/new/kraken_http_api/project.clj:21]: OK, let's see that PR then! :)

**Review by cap10morgan — APPROVED** [review]

**Comment by cap10morgan** [comment]
@tie-rack Do you want to merge this, do a new one for kehaar 0.10.1, and a third one for updating the other deps? Or something else?

**Comment by tie-rack** [comment]
@cap10morgan I'll update kehaar to 0.10.1 in this PR and make one for dep updates.

**Comment by cap10morgan** [comment]
@tie-rack Can you do the same for the `-works` one too?

**Comment by cap10morgan** [comment]
@tie-rack What should we do with this guy?

**Comment by tie-rack** [comment]
@cap10morgan even this is behind the times now, isn't it? Let's close and make a story to update these templates more generally. Sound okay?

**Comment by cap10morgan** [comment]
👍

**Pull request closed without merging**